src/java/org/apache/nutch/scoring/webgraph/NodeDumper.java

```diff
@@ -159,31 +181,17 @@ public void reduce(FloatWritable key, Iterator<Text> values,
    * of either the number of inlinks, the number of outlinks or the score. The
    * computed value is then either the sum of all parts or the top value.
    */
-  public static class Dumper extends Configured implements
-      Mapper<Text, Node, Text, FloatWritable>,
-      Reducer<Text, FloatWritable, Text, FloatWritable> {
-
-    private JobConf conf;
-    private boolean inlinks = false;
-    private boolean outlinks = false;
-    private boolean scores = false;
-    private long topn = Long.MAX_VALUE;
-    private boolean host = false;
-    private boolean domain = false;
-    private boolean sum = false;
-    private boolean max = false;
-
-    public void configure(JobConf conf) {
-      this.conf = conf;
-      this.inlinks = conf.getBoolean("inlinks", false);
-      this.outlinks = conf.getBoolean("outlinks", false);
-      this.scores = conf.getBoolean("scores", true);
-      this.topn = conf.getLong("topn", Long.MAX_VALUE);
-      this.host = conf.getBoolean("host", false);
-      this.domain = conf.getBoolean("domain", false);
-      this.sum = conf.getBoolean("sum", false);
-      this.max = conf.getBoolean("max", false);
-    }
+  public static class Dumper extends Configured {
+
+    private static Configuration conf;
+    private static boolean inlinks = false;
+    private static boolean outlinks = false;
+    private static boolean scores = false;
+    private static long topn = Long.MAX_VALUE;
+    private static boolean host = false;
+    private static boolean domain = false;
+    private static boolean sum = false;
+    private static boolean max = false;
```

Comment:
See comments in Generator etc. regarding static variables shared between mapper and reducer classes.